src/home/apis/getNews.ts

```diff
@@ -0,0 +1,7 @@
+import { searchClient } from '@/apis';
+import { SsuNews } from '@/home/types/news.type';
+
+export const getNews = async (): Promise<SsuNews> => {
+  const { data } = await searchClient.get('/search/news?platform=WEB');
+  return data;
```

Comment:
```suggestion
  const { data } = await searchClient.get('/search/news', {
    params: { platform: 'WEB' },
  });
```
다른 API 호출 코드들과 통일성을 맞추기 위해 `params` 객체로 search param을 전달하면 어떨까용